Kitodo/src/test/java/org/kitodo/MockDatabase.java

```diff
@@ -114,6 +122,22 @@ private static IndexRestClient initializeIndexRestClient() {
         return restClient;
     }
 
+    private static String readMapping() {
+        JSONParser parser = new JSONParser();
+        JSONObject jsonObject = new JSONObject();
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+        InputStream inputStream = classloader.getResourceAsStream("mapping.json");
```

Comment:
Opening a stream without closing it - this should be avoided and be replaced by try-with-resources statements.